File: Platform/codesignal/Intro/28.alphabeticShift/solution.js
function solution(s){
  return s.split("").map(x => (x.charCodeAt(0) > 121) ? 'a' : String.fromCharCode(x.charCodeAt(0) + 1)).join("")
}

function solution(s) {
  var c="abcdefghijklmnopqrstuvwxyza"
  return s.replace(/./g,x=>c[c.indexOf(x)+1])
}

console.log(solution("crazy"));

File: Platform/codesignal/Intro/28.alphabeticShift/solution.go
package main

import "fmt"

func solution(s string) string {
	res := ""
	for _, c := range s {
		if c == 'z' {
			res += "a"
		} else {
			res += string(c + 1)
		}
	}
	return res
}

func main() {
	s := "crazy"
	fmt.Println(s)
}
